### PR TITLE
Feat/dao 747 Write methods for getting NFT details used in table/card view

### PR DESCRIPTION
**Pull request by LordCheta**

### CODE REVIEW

File: .env.dao
Comment (sleyter93) [inline .env.dao:1]: We should not delete this file.

File: src/index.ts
Comment (sleyter93) [inline src/index.ts:1]: We should not delete this file.

File: src/blockscoutApi/index.ts [hunk 1]
```diff
@@ -169,4 +172,31 @@ export class BlockscoutAPI extends DataSource {
       .then(({ data }) => data.result)
       .catch(() => [])
   }
+
+  async getNftHoldersData ({ address, nextPageParams }: GetNftHoldersData) {
+    const url = `${this.url}/v2/tokens/${address.toLowerCase()}/instances`
+    console.log('getting nft holders data')
```
Comment (sleyter93) [inline src/blockscoutApi/index.ts]: Do you think this message is essential?

File: src/blockscoutApi/index.ts [hunk 2]
```diff
@@ -169,4 +172,31 @@ export class BlockscoutAPI extends DataSource {
...
+    try {
+      const response = await this.axios?.get<ServerResponseV2<NftTokenHoldersResponse>>(url, {
+        params: nextPageParams
+      })
+
+      if (response?.status === 200) {
+        const transformedResponse = nftHolderTransformedData(response.data.items)
```
Comment (sleyter93) [inline src/blockscoutApi/index.ts]: Maybe we could rename this; another option could be mapToNftHolder
```suggestion
        const nftHolders = transformResponseToNftHolder(response.data.items)
```

File: src/blockscoutApi/types.ts [hunk 3]
```diff
@@ -293,3 +293,48 @@ export interface BlockscoutTransactionResponseTxResult {
   txreceipt_status: string;
   value: string;
 }
+
+export interface NftTokenHolderOwner {
```
Comment (sleyter93) [inline src/blockscoutApi/types.ts]: We should reuse the Account interface, according to Blockscout docs, is reusing that interface.

**Review by sleyter93 — CHANGES_REQUESTED** [review]
Review the comments, and also, we need to include documentation for this new endpoint, you should add into `src/api/openapi.js`

File: .env.dao
Comment (LordCheta) [reply .env.dao:1]: Oh yes, I only removed from my commit, I made a change there only for my local development

File: src/index.ts
Comment (LordCheta) [reply src/index.ts:1]: This too

**Review by sleyter93 — APPROVED** [review]

**Pull request merged**